src/plugins/tables/tables.js

```diff
@@ -173,7 +174,12 @@ $document.on( "draw.dt", selector, function( event, settings ) {
 	// Identify that the table has been updated
 	$elm.trigger( "wb-updated" + selector, [ settings ] );
 } );
-
+$document.on( "keypress", ".paginate_button", function( event ) {
+	var $elm = $( event.target );
+	if (event.keyCode == 13){
```

Comment:
The space key is also use for navigation, please add it.

Also, I know you need to do this override because of how the datatable manage the pagination, please add a comment before that function explaining the rational.

Reply:
Ignore my comment about the space key behaviour. It should behave as any other links, sorry for the confusion.

Reply:
We have discussed this topic.